cmd/gossamer/import_runtime.go

```diff
@@ -29,7 +29,7 @@ func createGenesisWithRuntime(fp string) (string, error) {
 		return "", err
 	}
 
-	genesis.Genesis.Runtime["System"]["code"] = fmt.Sprintf("0x%x", runtime)
+	genesis.Genesis.Runtime["system"]["code"] = fmt.Sprintf("0x%x", runtime)
```

Comment:
Out of curiosity, was this right before? Or did this change with newer runtimes?

Reply:
this got changed in the new runtimes.  It would be nice if we marshaled this into a struct, and the json package would be able to interpret both capitalised and lowercase attribute names

Reply:
yeah we can also have `UnmarshalJSON` methods and the json package would use those directly. Although how should we marshal them? Always with lowercase `system` or would that depend on the runtime version 😕 


@danforbes can you create an issue for this, to have all fields of the genesis as struct and have this runtime field with custom MarshalJSON/UnmarshalJSON?